test/change-stream.test.js

```diff
@@ -0,0 +1,87 @@
+describe('PersistedModel.createChangeStream()', function() {
+  describe('configured to source changes locally', function() {
+    before(function() {
+      var test = this;
+      var app = loopback({localRegistry: true});
+      var ds = app.dataSource('ds', {connector: 'memory'});
+      this.Score = app.model('Score', {
+        dataSource: 'ds',
+        changeDataSource: false // use only local observers
+      });
+    });
+
+    it('should detect create', function(done) {
+      var Score = this.Score;
+
+      Score.createChangeStream(function(err, changes) {
+        changes.on('data', function(change) {
+          expect(change.type).to.equal('create');
+          changes.destroy();
+          done();
+        });
+
+        Score.create({team: 'foo'});
+      });
+    });
+
+    it('should detect update', function(done) {
+      var Score = this.Score;
+      Score.create({team: 'foo'}, function(err, newScore) {
+        Score.createChangeStream(function(err, changes) {
+          changes.on('data', function(change) {
+            expect(change.type).to.equal('update');
+            changes.destroy();
```

Comment:
`changes.destroy` is not called in case the assertion fails (throws an error). I think you should call the destroy method first, before making any assertions. Please fix this in the other unit-tests too.